plugins/delphes/examples/higgs_recoil_plots.C

```diff
@@ -0,0 +1,133 @@
+#ifndef __CINT__ 
+#include "edm4hep/MCParticleCollection.h"
+#include "edm4hep/RecoParticleRefCollection.h"
+#include "edm4hep/ReconstructedParticleCollection.h"
+#endif
+
+#include "podio/EventStore.h"
+#include "podio/ROOTReader.h"
+
+#include "TH1F.h"
+#include "TLorentzVector.h"
+
+/*
+put this into your .rootlogon.C file
+
+{
+ gInterpreter->AddIncludePath("$PODIO/include");
+ gSystem->Load("$PODIO/lib/libpodio.so");
+ gSystem->Load("$PODIO/lib/libpodioDict.so");
+ gSystem->Load("$PODIO/lib/libpodioRootIO.so");
+ gSystem->Load("$EDM4HEP/lib/libedm4hep.dylib");
+ gSystem->Load("$EDM4HEP/lib/libedm4hepDict.dylib");
```

Comment:
Are these endings mac specific for these libs?
Could we add these lines to the README?

Reply:
I think you can remove the endings, root will try to append `[.so | .dll | .dylib | .sl | .dl | .a] ` if nothing is given. Also, the absolute path is not necessary - the libraries just need to be installed to `LD_LIBRARY_PATH`. 

I'm not sure we can rely on `$PODIO` and `$EDM4HEP` being set  - they are definitely not set in the spack build.

Reply:
Ah I didn't know that root tries to resolve these automatically. In any case the [README](https://github.com/key4hep/EDM4hep/blob/master/plugins/delphes/examples/README.md) of the delphes examples mentions the requirements for the `LD_LIBRARY_PATH`.

Maybe loading all the libraries could be placed directly in this example as it is done for the other examples?